yt_dlp/extractor/niconico.py

```diff
@@ -1043,6 +1046,11 @@ def _real_extract(self, url):
                     **res,
                 })
 
+        for cookie in cookies:
+            self._set_cookie(
+                cookie['domain'], cookie['name'], cookie['value'],
+                expire_time=unified_timestamp(cookie['expires']), path=cookie['path'], secure=cookie['secure'])
```

Comment:
Is the `cookie['expires']` value not already a unix timestamp? IF it is, you can just use `int_or_none` instead of `unified_timestamp`